mlflow/utils/file_utils.py

```diff
@@ -21,11 +21,13 @@
 except ImportError:
     from yaml import SafeLoader as YamlSafeLoader, SafeDumper as YamlSafeDumper
 
+
 from mlflow.entities import FileInfo
 from mlflow.exceptions import MissingConfigException
 from mlflow.utils.rest_utils import cloud_storage_http_request, augmented_raise_for_status
 
 ENCODING = "utf-8"
+CODE = "code"
```

Comment:
Following from https://github.com/mlflow/mlflow/pull/5448/files#r822044736, let's move this to `model_utils` and use a clearer variable name, e.g. `FLAVOR_CONFIG_CODE`